docs/building-spark.md

```diff
@@ -153,7 +153,8 @@ Thus, the full flow for running continuous-compilation of the `core` submodule m
 
 # Using With IntelliJ IDEA
```

Comment:
I actually changed the wiki entry a bit. Maybe we could have a header that says

```
# Building Spark with IntelliJ IDEA or Eclipse
```

And it could refer to our IDE section:

https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark#ContributingtoSpark-IDESetup